src/test/java/de/rabea/controller/LogsControllerTest.java

```diff
@@ -14,22 +20,34 @@
 
 public class LogsControllerTest {
 
+    private String pathToEmptyFile;
+
+    @Rule
+    public TemporaryFolder folder = new TemporaryFolder();
+
+    @Before
+    public void setup() throws IOException {
+        File file = folder.newFile("log.txt");
+        pathToEmptyFile = file.getAbsolutePath();
+    }
+
     @Test
     public void returnsResponseWithLogsIfAuthorised() {
-        Logger logger = new Logger();
-        logger.log(new RequestLine("GET /somewhere HTTP/1.1"));
+        MultiLogger logger = new MultiLogger();
+        logger.add(new FileLogger(pathToEmptyFile));
```

Comment:
@felipesere
You have to know that you need to add different loggers to the `MultiLogger`, both in the tests and also in Main. Is that just how the pattern works or are there any ways to make it more obvious that other loggers need to be added?